src/ResourceManager/AnalysisServices/Commands.AnalysisServices/help/New-AzureRmAnalysisServicesFirewallConfig.md

```diff
@@ -0,0 +1,71 @@
+﻿---
+external help file: Microsoft.Azure.Commands.AnalysisServices.dll-Help.xml
+Module Name: AzureRM.AnalysisServices
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.analysisservices/new-azurermanalysisservicesfirewallconfig
+schema: 2.0.0
+---
+
+# New-AzureRmAnalysisServicesFirewallConfig
+
+## SYNOPSIS
+Creates a new Analysis Services firewall config 
+
+## SYNTAX
+
+```
+New-AzureRmAnalysisServicesFirewallConfig [-EnablePowerBIService] <Boolean> [-FirewallRules] <list of firewall rule> 
+```
+
+## DESCRIPTION
+The New-AzureRmAnalysisServicesFirewallConfig creates a new firewall config object
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> New-AzureRmAnalysisServicesFirewallConfig -EnablePowerBIService $False -FirewallRules $TheFirewallRules
```

Comment:
Should have the actual cmdlet to create "TheFirewallRules" - also fix EnablePowerBIService once you switch is to SwitchParameter.

Reply:
done